src/main/java/se/kth/sda/skeleton/profile/Profile.java

```diff
@@ -0,0 +1,91 @@
+package se.kth.sda.skeleton.profile;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import javax.persistence.*;
+
+import com.fasterxml.jackson.annotation.*;
+
+import se.kth.sda.skeleton.user.User;
+
+@Entity
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+public class Profile {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String displayDescription;
+
+    @Column(nullable = false)
+    private String displayName;
+
+    @Column(nullable = false)
+    private Boolean activeProfile;
```

Comment:
Could also add `@NotBlank `annotation for every time you use `@Column(nullable = false)` for extra checking

Reply:
Added the annotation to the field